### ISSUE TITLE
landscape visibility mask stopped working?

### ISSUE THREAD

**Issue by Kettenotter** [sideeffects/HoudiniEngineForUnreal-v2#176]
I am using HoudiniEngine-v2.0.5-H18.5.759. Tested on UE4.26 and UE5

### Houdini Setup:

<img width="1171" alt="image" src="https://user-images.githubusercontent.com/32515271/151665554-ee714e51-b9c1-43d0-bca7-b26e364fee07.png">

### Landscape Material:

<img width="662" alt="image" src="https://user-images.githubusercontent.com/32515271/151665688-b1699d91-52c8-4f66-8923-634af7a2eba0.png">

Visibility mask **will not be importet from houdini**, I am still able to paint by Hand:

<img width="1210" alt="image" src="https://user-images.githubusercontent.com/32515271/151665737-59e7f3ae-10f9-45d6-b1ed-a65ec354acf2.png">

Anyone got landscape visbility mask from houdini working or is this a bug? (In an older version it was working for me)

**Comment by faxlab** [comment]
> I am using HoudiniEngine-v2.0.5-H18.5.759. Tested on UE4.26 and UE5
> 
> ### Houdini Setup:
> <img alt="image" width="1171" src="https://user-images.githubusercontent.com/32515271/151665554-ee714e51-b9c1-43d0-bca7-b26e364fee07.png">
> 
> ### Landscape Material:
> <img alt="image" width="662" src="https://user-images.githubusercontent.com/32515271/151665688-b1699d91-52c8-4f66-8923-634af7a2eba0.png">
> 
> Visibility mask **will not be importet from houdini**, I am still able to paint by Hand:
> 
> <img alt="image" width="1210" src="https://user-images.githubusercontent.com/32515271/151665737-59e7f3ae-10f9-45d6-b1ed-a65ec354acf2.png">
> 
> Anyone got landscape visbility mask from houdini working or is this a bug? (In an older version it was working for me)

Hi. Did you ever solve this? I can't get the visibility mask to work. Im getting same black parts and not actual visibility layer. Thanks for any tips and info!

**Comment by Kettenotter** [comment]
@faxcorp 
I don't think so. But I remember that it did work before...? But it's some time ago that I played around with visibility masks.

But I also just exported the visibility mask from Houdini as a texture and imported it as a visibility mask by hand. Which worked too. Just had to make some adjustments to fix rotation and stuff. Which can be done in Houdini.
(If you go that path also look up the unreal supported landscape resolutions. Might help with alignment)

**Comment by faxlab** [comment]
Oh great workaround! Thanks very much for sharing!
